R/globals.R

```diff
@@ -3,4 +3,4 @@ utils::globalVariables(c(".","Mean(R)","Quantile.0.025(R)","Quantile.0.975(R)",
                          "inc.deconvol","iteration","lo","lwr","lwr.agg",
                          "mean.agg","mean.agg.reldiff","name","obs","trash",
                          "upr","upr.agg","use","use.cumm","val","val_smooth",
-                         "value","y"))
+                         "value","y","aes","geom_line","silent","x"))
```

Comment:
i'm not sure why `aes` and `geom_line` are declared here as global variables? if this was added to clear notes in `devtools::check()` about undeclared global variables, i think the fix is to ensure these functions are always referred to with the package name, _i.e._ `ggplot2::aes` and ggplot2::geom_line()`. 

i've done a global search through the package package code on this branch for any bare `aes()` and `geom_line()` calles, and prefixed them with `ggplot2::` (there was only one such instance of each). i've also removed `aes` from `geom_line` from `globals.R`. this is all in commit 789ec7f and has been pushed to this branch on `origin`.

Reply:
(so no further action needs to be taken to resolve this comment)